apps/aefate/src/aefa_fate_op.erl

```diff
@@ -178,30 +178,30 @@ jumpif(Arg0, Arg1, EngineState) ->
 switch(Arg0, Arg1, Arg2, EngineState) ->
     {Value, ES1} = get_op_arg(Arg0, EngineState),
     if ?IS_FATE_VARIANT(Value) ->
-            ?FATE_VARIANT(Size, Tag, _T) = Value,
-            if Size =:= 2 ->
+            ?FATE_VARIANT(Arities, Tag, _T) = Value,
+            if length(Arities) =:= 2 ->
                     case Tag of
                         0 -> {jump, Arg1, ES1};
                         1 -> {jump, Arg2, ES1};
-                        _ -> aefa_fate:abort({bad_variant_tag, Tag, Size}, ES1)
+                        _ -> aefa_fate:abort({bad_variant_tag, Tag}, ES1)
```

Comment:
This change does not seem to be handled in `aefa_fate:abort/2`

Reply:
Good catch, had it on my to do in the head, obviously not in a test case.

Reply:
This can not happen unless you manage to create variant data that is inconsistent in it self, which should not be possible. Ill come up with a fix.